static/css/introjs.scss

```diff
@@ -0,0 +1,347 @@
+.introjs-overlay {
```

Comment:
Sure.  There is a special rule for addon/data/vendor that just copies everything with no transformation, which we could use for static/vendor too.